File: dataset/output/MISTRAL_codestral-latest/Task133_MISTRAL_codestral-latest.c
// C
#include <stdio.h>
#include <string.h>

#define MAX_PASSWORD_LENGTH 50

int main() {
    char currentPassword[MAX_PASSWORD_LENGTH], newPassword[MAX_PASSWORD_LENGTH], confirmPassword[MAX_PASSWORD_LENGTH];

    printf("Enter your current password: ");
    fgets(currentPassword, MAX_PASSWORD_LENGTH, stdin);
    currentPassword[strcspn(currentPassword, "
")] = 0; // Remove trailing newline

    printf("Enter your new password: ");
    fgets(newPassword, MAX_PASSWORD_LENGTH, stdin);
    newPassword[strcspn(newPassword, "
")] = 0; // Remove trailing newline

    printf("Confirm your new password: ");
    fgets(confirmPassword, MAX_PASSWORD_LENGTH, stdin);
    confirmPassword[strcspn(confirmPassword, "
")] = 0; // Remove trailing newline

    if (strcmp(newPassword, confirmPassword) == 0) {
        printf("Password reset successful.
");
    } else {
        printf("Passwords do not match. Password reset failed.
");
    }

    return 0;
}

File: dataset/output-bk/MISTRAL_codestral-latest/Task133_MISTRAL_codestral-latest.java
// Java
import java.util.Scanner;

public class PasswordReset {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        System.out.print("Enter your current password: ");
        String currentPassword = scanner.nextLine();

        System.out.print("Enter your new password: ");
        String newPassword = scanner.nextLine();

        System.out.print("Confirm your new password: ");
        String confirmPassword = scanner.nextLine();

        if (newPassword.equals(confirmPassword)) {
            System.out.println("Password reset successful.");
        } else {
            System.out.println("Passwords do not match. Password reset failed.");
        }
    }
}